### ISSUE TITLE
tensorflow.example does not work with TF 2.4.1 -- are you using TF 1.6.1 for a specific reason?

### ISSUE THREAD

**Issue by katjahauser** [BorgwardtLab/Neural-Persistence#1]
Hey there, 

in batchnorm_simple.py line 7 you are importing `from tensorflow.examples.tutorials.mnist import input_data` . This does not work with Tensorflow 2.4.1 anymore. The correct call with 2.4.1. is `tensorflow.keras.datasets.mnist` afaik. 

I am opening an issue, because I don't know whether you are using TF 1.6.1 for a specific reason. 

Best regards, 
Katja

**Comment by katjahauser** [comment]
I came up with a solution. Sorry for the bother.

**Comment by Pseudomanifold** [comment]
What's the solution? Happy to update the code :-)

**Comment by katjahauser** [comment]
Frankly speaking, I ended up using another experimental framework all-together and now only need your persistence calculations ^^, Since tda.PerLayerCalculation does not require TF, but only an OrderedDict of the appropriate format (which was a good design choice, imho), I simply wrote a converter (model->OrderedDict).
